.github/workflows/docsify-cli.yml

```diff
@@ -0,0 +1,27 @@
+name: docsify-cli
+
+on: [ push, pull_request ]
```

Comment:
Let's have the [workflow_dispatch](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_dispatch) event as well.

Reply:
What needs is this used to fulfill?

Reply:
It allows us to trigger workflows manually.